Resources/doc/features/tagging.rst

```diff
@@ -76,6 +76,29 @@ To invalidate tags, call ``TagHandler::invalidateTags($tags)``::
 
 See the :ref:`Tag Handler reference <tag_handler_addtags>` for full details.
 
+Tagging from Twig Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In situations where a page is assembled in the templating layer, it can be more
+convenient to add tags from inside the template. This works the same way as
+with the tag handler and can also be mixed with the other methods:
+
+.. code-block:: jinja
+
+    {# template.html.twig #}
+    {{ fos_httpcache_tag('mytag') }}
+    {{ fos_httpcache_tag(['tag-one', 'tag-two']) }}
+
+.. hint::
+
+    This twig function never outputs anything into the template but is only
+    called for the side effect of adding the tag to the response header.
```

Comment:
yeah its a bit of an edge case. its also not really the job of the templating to control tagging, but i feel it will be useful in some scenarios.